lib/osf-components/addon/components/registries/partial-registration-modal/template.hbs

```diff
@@ -8,6 +8,9 @@
         {{t 'registries.partialRegistrationModal.title'}}
     </dialog.heading>
     <dialog.main>
+        <div local-class='component-contributor-warning'>
```

Comment:
What is this css class for? If it's not currently used, should we add a `max-width` to it so that the modal's width doesn't expand to accommodate the text?

Reply:
It's not used; I added it just in case. We could add a max-width if we want. Is that something we do elsewhere with dialog boxes?

Reply:
One example I found is in `finalize-registration-modal/template.hbs` where we put a `max-width: 540px` on the `div` for the notice text so that the text doesn't expand the dialog to take up the entire screen.